mars/dataframe/groupby/tests/test_groupby.py

```diff
@@ -62,6 +62,10 @@ def testGroupBy(self):
         with self.assertRaises(TypeError):
             ms.groupby(lambda x: x + 1, as_index=False)
 
+        df = md.DataFrame(pd.DataFrame(np.random.rand(100, 3), columns=list('abc')))
+        grouped = df.groupby(['a', 'b'])
+        self.assertIn('DataFrameGroupBy', repr(grouped))
```

Comment:
This unittest is wrong, if grouped is not executed, the error mentioned in #1413 would not happen.